cfn-resources/stream-connection/Makefile

```diff
@@ -17,3 +17,17 @@ debug:
 
 clean:
 	rm -rf bin
+
+create-test-resources:
+	@echo "==> Creating test files for contract testing"
+	./test/contract-testing/cfn-test-create-inputs.sh
+
+delete-test-resources:
+	@echo "==> Delete test resources used for contract testing"
+	./test/cfn-test-delete-inputs.sh
+
+run-contract-testing:
+	@echo "==> Run contract testing"
+	make build
+	sam local start-lambda &
+	cfn test --function-name TestEntrypoint --verbose
```

Comment:
have last line in the file with new line

Reply:
updated